src/outputUtils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Parse column based output which is seperated by whitespace(s) from kubectl or similar sources
+ * for example, kubectl get po
+ * @param lineOutput raw output with headers from kubectl or similar sources
+ * @param parsingRegex a regex for the column separators
```

Comment:
I'd prefer to change the name of this argument to something like `columnSeparator` though it's not a showstopper.

Reply:
Changed name.